System/Math.scala

```diff
@@ -0,0 +1,54 @@
+package System
+
+object Math
```

Comment:
This matches C# default MidpointRounding.ToEven. - However this code requires changes in converter which is not in this commit. (will do separately as we have other changes in converter)